src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.cs

```diff
@@ -525,28 +514,21 @@ private async Task<Compilation> GetOrBuildDeclarationCompilationAsync(SolutionSe
 
             private async Task<CompilationInfo> BuildCompilationInfoFromScratchAsync(
                 SolutionState solution,
-                TextDocumentStates<SourceGeneratedDocumentState>? authoritativeGeneratedDocuments,
-                TextDocumentStates<SourceGeneratedDocumentState> nonAuthoritativeGeneratedDocuments,
-                GeneratorDriver? generatorDriver,
+                CompilationTrackerGeneratorInfo generatorInfo,
                 CancellationToken cancellationToken)
             {
                 try
                 {
                     var compilation = await BuildDeclarationCompilationFromScratchAsync(
                         solution.Services,
-                        new CompilationTrackerGeneratorInfo(
-                            nonAuthoritativeGeneratedDocuments,
-                            generatorDriver,
-                            documentsAreFinal: false),
+                        generatorInfo,
```

Comment:
Previously `documentsAreFinal` was explicitly set to false, is this change intended?